scrapy/core/downloader/webclient.py

```diff
@@ -1,8 +1,8 @@
 import re
 from time import time
 from urllib.parse import urlparse, urlunparse, urldefrag
-
 from twisted.web.http import HTTPClient
+
 from twisted.internet import defer
```

Comment:
Some import sorting and grouping changes like this one look wrong to me, but I am OK with merging this as an overall improvement, and in a separate pull request introduce a CI job to check these with isort, and include isort into the pre-commit configuration that we may introduce in https://github.com/scrapy/scrapy/pull/5734.